src/meta/Meta.cmake

```diff
@@ -2,15 +2,15 @@ set(NEBULA_META NMeta)
 
 # build nebula.api library
 # target_include_directories(${NEBULA_META} INTERFACE src/meta)
-add_library(${NEBULA_META} STATIC 
-    ${NEBULA_SRC}/meta/ClusterInfo.cpp
-    ${NEBULA_SRC}/meta/NodeManager.cpp
-    ${NEBULA_SRC}/meta/Table.cpp
-    ${NEBULA_SRC}/meta/TestTable.cpp)
+add_library(${NEBULA_META} STATIC
+        ${NEBULA_SRC}/meta/ClusterInfo.cpp
+        ${NEBULA_SRC}/meta/NodeManager.cpp
+        ${NEBULA_SRC}/meta/Table.cpp
+        ${NEBULA_SRC}/meta/TestTable.cpp)
```

Comment:
hmm - the indentation doesn't look correct, could you please update tab size to 4 rather than 8 in your IDE and reformat this?